src/pages/iou/request/step/IOURequestEditReport.tsx

```diff
@@ -35,11 +38,21 @@ function IOURequestEditReport({route}: IOURequestEditReportProps) {
         Navigation.dismissModalWithReport({reportID: item.value});
     };
 
+    const removeFromReport = () => {
+        if (!transactionReport || selectedTransactionsID.length === 0) {
+            return;
+        }
+        changeTransactionsReport(selectedTransactionsID, CONST.REPORT.UNREPORTED_REPORT_ID);
+        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(selfDMReportID));
```

Comment:
I think here we want to just navigate the user back to where they came from, so either dismiss the modal or navigate back to `backTo`. We don't really wanna navigate them to the selfDM